### PR TITLE
Upgrade for lab 3.0 as a prebuilt extension

### PR DESCRIPTION
**Pull request by mlucool**
This commit was partially generated via juptyerlab.upgrade_extension

### CODE REVIEW

**Comment by ian-whitestone** [comment]
@mlucool if you can advise on how to install an extension from a branch, i can test this out

**Comment by LunarLanding** [comment]
@ian-whitestone download the repo jlab3 ( https://github.com/deshaw/jupyterlab-execute-time/archive/jlab3.zip )
unzip, cd to the repo, and 'pip install .'
@mlucool works for me! hopefully this can be put on conda-forge, now that it is a python package.

**Comment by ian-whitestone** [comment]
thanks @LunarLanding! installation worked (see installed packages in left of screenshot), but i'm not getting the expected output with the "last executed at" and cell execution time:

image

Was there something else I needed to do to get it working? Already tried restarting jupyter lab

**Comment by mlucool** [comment]
I wonder if you setting didn't stick for some reason? Per the README:

> Note: for this to show anything, you need to enable cell timing in the notebook via Settings->Advanced Settings Editor->Notebook: {"recordTiming": true}. This is a notebook metadata setting and not a plugin setting. The plugin just displays this data.

**Comment by ian-whitestone** [comment]
> I wonder if your setting didn't stick form 2.0 to 3.0 some reason. Per the README:
> 
> > Note: for this to show anything, you need to enable cell timing in the notebook via Settings->Advanced Settings Editor->Notebook: {"recordTiming": true}. This is a notebook metadata setting and not a plugin setting. The plugin just displays this data.

ah that did it, thanks! 

✅ working for me now on jupyter lab v3.0.4

File: binder/environment.yml [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+name: jupyterlab_execute_time-demo
+
 channels:
   - conda-forge
+
 dependencies:
-  - jupyterlab>=2
-  - nodejs=12
+  # runtime dependencies
+  - python >=3.8,<3.9.0a0
```
Comment (late-warrior) [inline binder/environment.yml:14]: Not sure what the python breaking version convention is, but maybe this can be wider ? (<4.0.0 ?)

**Review by late-warrior — APPROVED** [review]
Minor comment inline, we can publish a beta version from this branch ?

File: binder/environment.yml (hunk 1 above)
Comment (mlucool) [reply binder/environment.yml:14]: This was generated, but I agree. Fixed.

**Pull request merged**